maintainers/maintainer-list.nix

```diff
@@ -15780,6 +15780,12 @@
       fingerprint = "B234 EFD4 2B42 FE81 EE4D  7627 F72C 4A88 7F9A 24CA";
     }];
   };
+  sironheart = {
```

Comment:
Again:

Split this in a separate commit (no need for a new PR).

Reply:
Didn't I do this with the second commit? Hash d86a1052a5956d80f6ee735dcba3631177c8da1c

Reply:
When I look the commit listing, I only see ` kdoctor: init at 1.0.1`. Four times.
Why?

Reply:
I usually use the first line just the PR head and any additional lines for further information. I could squash/ reword the commits, if you want.

Reply:
1. If you want to add yourself to the maintainers file:
   1. **Do not open a pull request for it**
   2. Use the same pull request you are opening for including a package
   3. In case of need, Google "how to split a pull request into two"
   4. The specific commit for adding yourself to the maintainers file should follow the exact template below:

   ```
   maintainers: add <handle>
   ```

2. When adding a new package, we do not need to know how many times you reviewed and fixed it before inclusion. 
   1. Squash all those modifications
   2. The specific commit for adding a new package should follow the template below:

   ```
   <package name>: init at <version>

   <optional short description>
   ```

Reply:
Fixed and reworded it. Sorry about that.